tools/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.props

```diff
@@ -27,7 +27,6 @@
       <PropertyGroup>
         <IceRpcProtocPath>$(MSBuildThisFileDirectory)../tools/</IceRpcProtocPath>
         <IceRpcProtocGenPath>$(MSBuildThisFileDirectory)../tools/</IceRpcProtocGenPath>
-        <IceRpcBuildTelemetryScriptPath>$(MSBuildThisFileDirectory)../tools/</IceRpcBuildTelemetryScriptPath>
```

Comment:
We need the other Path variables because they are different when building with sources and when building with the NuGet package. Build telemetry is always disabled for source builds, and no need to use variable for the path.